dbus/methods.go

```diff
@@ -598,3 +598,43 @@ func unitPath(name string) dbus.ObjectPath {
 func unitName(dpath dbus.ObjectPath) string {
 	return pathBusUnescape(path.Base(string(dpath)))
 }
+
+type JobStatus struct {
+	Id       int
+	Unit     string
+	JobType  string
+	Status   string
+	JobPath  string
+	UnitPath string
+}
+
+// ListJobs returns an array with all currently queued jobs
+func (c *Conn) ListJobs() ([]JobStatus, error) {
+	return c.listJobsInternal(c.sysobj.Call("org.freedesktop.systemd1.Manager.ListJobs", 0).Store)
+}
+
+func (c *Conn) listJobsInternal(f storeFunc) ([]JobStatus, error) {
+	result := make([][]interface{}, 0)
+	err := f(&result)
+	if err != nil {
```

Comment:
I think you can merge this with the above call: `if err:= foo(); err != nil { .. }`

Reply:
Done